helm/trivy/templates/configmap.yaml

```diff
@@ -9,6 +9,7 @@ data:
   TRIVY_CACHE_DIR: "/home/scanner/.cache/trivy"
 {{- if .Values.trivy.cache.redis.enabled }}
   TRIVY_CACHE_BACKEND: {{ .Values.trivy.cache.redis.url | quote }}
+  TRIVY_CACHE_TTL: {{ .Values.trivy.cache.redis.ttl | quote }}
```

Comment:
Please update the [readme- parameters section](https://github.com/aquasecurity/trivy/blob/main/helm/trivy/README.md)

| `trivy.cache.redis.ttl`               | Specify redis ttl. e.g 3600s, 24h

Reply:
Thanks, I forgot about that!